src/OpenLoco/src/Objects/Object.h

```diff
@@ -104,13 +99,19 @@ namespace OpenLoco
             }
         }
 
-        bool operator!=(const ObjectHeader& rhs) const
+        constexpr bool operator!=(const ObjectHeader& rhs) const
         {
             return !(*this == rhs);
         }
     };
     static_assert(sizeof(ObjectHeader) == 0x10);
 #pragma pack(pop)
+    constexpr ObjectHeader kEmptyObjectHeader = ObjectHeader{ 0xFFFFFFFFU, { '\xFF', '\xFF', '\xFF', '\xFF', '\xFF', '\xFF', '\xFF', '\xFF' }, 0xFFFFFFFFU };
+
+    constexpr bool ObjectHeader::isEmpty() const
+    {
+        return *this == kEmptyObjectHeader;
```

Comment:
might be better to just make this a memcmp rather than using == as isCustom shouldn't really be checked.